src/bridge/mod.rs

```diff
@@ -85,6 +85,7 @@ async fn start_neovim_runtime() {
     let nvim = Arc::new(nvim);
 
     start_ui_command_handler(nvim.clone());
+
```

Comment:
I assume this is an accidental leftover, right?

Reply:
(It's nothing breaking, but let's keep the commits/changes tidy)

Reply:
Of course, no problem